scripts/gogetcookie.sh

```diff
@@ -1,10 +1,9 @@
 #!/bin/bash
 
-touch ~/.gitcookies
-chmod 0600 ~/.gitcookies
+ touch ~/.gitcookies
+ chmod 0600 ~/.gitcookies
```

Comment:
Can you check the whitespaces here and the rest of this file please?

Reply:
I will do that.